Main/Main.js

```diff
@@ -42,14 +90,18 @@ const Main = ({
         toCurrency={toCurrency}
         onChangetoCurrency={onChangetoCurrency}
       />
+       <FlatButton text="Exchange Currency!" onPress={pressHandler} />
```

Comment:
I think we should rename the button text. I have two suggestions: 
1) more "professional":
- simple _Exchange_
2) a bit "funnier":
- _Show me the money!_ --> it has literal sense to show the currency exchange amount but also relates to you! because in South Korea, as you probably now, there is a tv show with the same title. the show relates to rap though 😂. but I still think it's funny

Reply:
Agree, I know the show me the money tv show! haha this is good idea, let's do this!!! 
We can discuss about this in today's meeting too!!